airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -95,7 +103,17 @@
     delete_report = GoogleDisplayVideo360DeleteReportOperator(
         report_id=report_id, task_id="delete_report"
     )
-    # # [END howto_google_display_video_deletequery_report_operator]
+    # [END howto_google_display_video_deletequery_report_operator]
+
+    # [START howto_google_display_video_download_line_items_operator]
+    download_line_items = GoogleDisplayVideo360DownloadLineItemsOperator(
+        task_id="download_line_items",
+        request_body=REQUEST_BODY,
```

Comment:
I am not sure about this `request_body`, it doesn't sound best from user experience I think. I know that it is not guide-compliant but should we pass keys from `REQUEST_BODY` as arguments to the operator? @potiuk @mik-laj

Reply:
I don't know and don't have strong opinion here. I think we discussed about it. For me personally passing request is more handy then passing all arguments. My first version had arguments, but Kamil said that body request is better for the users.

Reply:
I think we all should stick to the same convention to avoid different points of view.